aldryn_forms/admin.py

```diff
@@ -20,10 +21,33 @@
 class FormDataAdmin(admin.ModelAdmin):
 
     date_hierarchy = 'sent_at'
-    list_display = ['__unicode__', 'sent_at']
-    list_filter = ['name', 'people_notified']
+    list_display = ['__unicode__', 'sent_at', 'language']
+    list_filter = ['name', 'language']
     model = FormData
-    readonly_fields = ['name', 'data', 'language', 'sent_at', 'people_notified']
+    readonly_fields = [
+        'name',
+        'data',
+        'language',
+        'sent_at',
+        'get_people_notified'
+    ]
+
+    def has_add_permission(self, request):
+        return False
+
+    def get_people_notified(self, obj):
+        people_list = obj.people_notified.split(':::')
```

Comment:
people_notified value could be Paulo <paulo.alvarado@divio.ch> or paulo.alvarado@divio.ch but there could be many of these when a form is submitted, so I had to delimit each set by something. Given that we use ```:``` as a field delimiter already, I figured I would just use ```:::``` to avoid any conflicts.. thought about using newlines but I'm not really a fan of newlines for delimiting.. specially user driven content.